aqt/archives.py

```diff
@@ -425,8 +425,8 @@ def _get_archives_base(self, name, target_packages):
 
         if self.version >= Version("6.8.0"):
             arch = self.arch
-            if self.os_name == "windows":
-                arch = self.arch.replace("win64_", "", 1)
+            if self.os_name == "windows" or self.os_name == "windows_arm64":
+                arch = self.arch.replace("win64_", "", 1).replace("_cross_compiled", "", 1)
```

Comment:
```suggestion
                arch = self.arch.replace("win64_", "", 1)
                if self.os_name == "windows":
                    arch = self.arch.replace("_cross_compiled", "", 1)
```

Reply:
A slightly different fix is in 4a53344c99a2db45c06e62e3c39510dcabc53ca7